imports/libs/server/cron/initial_email_notifications.js

```diff
@@ -0,0 +1,268 @@
+import { CBMailer } from '/imports/libs/server/cb_mailer/';
+import { Meteor } from 'meteor/meteor';
+import _ from 'lodash';
+import moment from 'moment';
+import {
+  getEligibleRecipients,
+  getGroupMembers,
+  getGroupOrganizers,
+  getDiscussionSubscribers,
+  getNewHangouts,
+  getNewDiscussions,
+  getNewDiscussionResponses,
+  getNewMembers,
+  createdFromNow } from './helpers';
+
+
+/**
+* send email notification for new hangouts to eligible recipients
+* @function
+* @name initialEmailNotificationsForHangouts
+*/
+async function initialEmailNotificationsForHangouts() {
+   let hangouts = await getNewHangouts();
```

Comment:
maybe this is worth wrapping in try catch.

Reply:
`getNewHangouts` doesn't throw any err, So I'm not sure what purpose would it serve.